wscript_build.py

```diff
@@ -127,6 +127,18 @@ def build(ctx):
         ctx.wayland_protocol_header(proto_dir = ctx.env.WL_PROTO_DIR,
             protocol  = "unstable/xdg-decoration/xdg-decoration-unstable-v1",
             target    = "generated/wayland/xdg-decoration-unstable-v1.h")
+        ctx.wayland_protocol_code(proto_dir = ctx.env.WL_PROTO_DIR,
+            protocol  = "unstable/linux-dmabuf/linux-dmabuf-unstable-v1",
+            target    = "generated/wayland/linux-dmabuf-unstable-v1.c")
+        ctx.wayland_protocol_header(proto_dir = ctx.env.WL_PROTO_DIR,
+            protocol  = "unstable/linux-dmabuf/linux-dmabuf-unstable-v1",
```

Comment:
do these unstable protocols require a newer version of wayland-protocols?
see https://github.com/mpv-player/mpv/blob/defb02daa461200f4de972c57c4ac8dc108feb5f/waftools/checks/custom.py#L103-L106